### PR TITLE
Update package-lock.json; let `mvn clean install` work on clean repo

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
If `node_modules` is not present, `mvn clean install` fails on main because it cannot find `locktt`. This PR allow `mvn clean install` when node_modules is absent.

Also updated the package-lock.json files caused by new version of locktt

### Referenced pull requests

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] tests</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaweb-employee-rostering] LTS</b>

</details>

### CODE REVIEW

File: optaweb-employee-rostering-frontend/pom.xml [hunk 1]
```diff
@@ -50,6 +50,16 @@
...
+          <execution>
+            <id>npm install lock-treatment-tool and run-node</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>npm</goal>
+            </goals>
+            <configuration>
+              <arguments>install @kie/lock-treatment-tool</arguments>
+            </configuration>
+          </execution>
```
Comment (yurloc) [inline optaweb-employee-rostering-frontend/pom.xml:62]: IDK... Is this neccessary? @Ginxo can't we move `locktt` completely to the `productized` profile as you did in https://github.com/kiegroup/optaweb-employee-rostering/pull/734?

Comment (Ginxo) [reply optaweb-employee-rostering-frontend/pom.xml:62]: well locktt is not needed by community (unless this is built with no access to npmjs registry) I think `productized` profile is the proper place to add locktt.

Comment (Ginxo) [reply optaweb-employee-rostering-frontend/pom.xml:62]: and this is additionally defined at #L66 by `<id>lock-treatment-tool execution</id>` , why is it needed again? (without `--`)

**Comment by triceo** [comment]
`optaweb-employee-rostering` project is no longer maintained, closing this PR.

**Pull request closed without merging**